### PR TITLE
Feature/boas 694 bulk download

### PR DESCRIPTION
**Pull request by lutamKC**
## Describe your changes

Implement bulk download. This should keep working whatever solution for the actual download we choose

## Type of change 🧩

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation update
- [ ] Other (please explain in the description section above)

## Checklist before requesting a review

- [ ] I have performed a self-review of my own code
- [ ] I have double checked this work does not include any hardcoded secrets or passwords
- [ ] I have made corresponding changes to the documentation
- [ ] I have provided details on how I have tested my code
- [ ] I have referenced the ticket number above
- [ ] I have provided a description of the ticket
- [ ] I have included unit tests to cover any testable code changes

### CODE REVIEW

File: apps/web/src/client/components/files-list/files-list.module.js [hunk 1]
```diff
@@ -43,7 +56,42 @@ const initFilesListModule = () => {
...
+		}
+
+		for (const selectedCheckbox of checkedFiles) {
+			const { value: selectedGuid } = selectedCheckbox;
+
+			/** @type {HTMLElement | null} */
+			const downloadLink = document.querySelector(`a[data-action='download-${selectedGuid}']`);
+
+			if (downloadLink) {
+				// system interrupts download requests if they all happen at the very same moment
```
Comment (lutamKC) [inline apps/web/src/client/components/files-list/files-list.module.js:84]: I tested this with a big file and still works, which means that it's ok to send a download request when other files are still being downloaded. Important thing is not to send all the requests at once.

File: apps/web/src/client/components/files-list/_html.js [hunk 2]
```diff
@@ -0,0 +1,25 @@
+/**
+ * Create error banner based on GovUk GDS html
+ *
+ * @param {string} errorMessage
+ * @returns {string}
+ */
+export const buildErrorBanner = (errorMessage) => {
+	return `
+	<div class="govuk-grid-row">
```
Comment (lutamKC) [inline apps/web/src/client/components/files-list/_html.js:9]: copy/paste of the GDS error banner

Comment (francescobrenci-AR) [reply apps/web/src/client/components/files-list/_html.js:9]: can you use the GDS macro?

**Review by francescobrenci-AR — CHANGES_REQUESTED** [review]

**Review by francescobrenci-AR — DISMISSED** [review]

**Review by francescobrenci-AR — APPROVED** [review]

**Pull request merged**